src/apps/external_executor/external_executor.cpp

```diff
@@ -17,11 +21,14 @@
 namespace externalexecutor
 {
   using Map = kv::Map<std::string, std::string>;
+  using ExecutorNodeId = ccf::EntityId<ccf::NodeIdFormatter>;
```

Comment:
`ccf::NodeId`?

Reply:
Yeah, it's the same. It's just a bit of future proofing -- in case we decide to use a custom `ExecutorNodeIdFormatter` instead of `NodeIdFormatter`